Migration.md

```diff
@@ -144,6 +144,7 @@ The following deprecated methods and classes have been removed.
 1. **sdf/InterfaceElements.hh**: The struct `NestedInclude` has been converted 
    to a class. Accessing data members directly is deprecated. Instead use the 
    corresponding member functions.
+2. The `ignition` namespace is deprecated and will be removed in future versions.  Use `gz` instead.
```

Comment:
This is under the 11 -> 12 migration, we need to create a new section for 12 -> 13.

Reply:
Rectified in force-push https://github.com/gazebosim/sdformat/pull/1022/commits/6883a17bb39d9aed8bcc3a1bd8b3ebc38731cae7